components/button/examples/50-active-state.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { Button } from '../src';
+
+export default () => (
+	<>
+		<p>
+			Focus styling is provided with a pseudo-class. The following have been programatically forced
+			via an <code>active</code> class.
+		</p>
+
+		<h3>
+			Active buttons with a <code>&lt;button&gt;</code> tag
+		</h3>
+		<p>
+			<Button appearance="primary" className="active">
+				Primary
+			</Button>{' '}
+			<Button appearance="hero" className="active">
+				Hero
+			</Button>{' '}
+			<Button appearance="neutral" className="active">
+				Neutral
+			</Button>{' '}
+			<Button appearance="faint" className="active">
+				Faint
+			</Button>
+		</p>
+		<p>
+			<Button appearance="primary" className="active" soft>
+				Primary soft
+			</Button>{' '}
+			<Button appearance="hero" className="active" soft>
+				Hero soft
+			</Button>{' '}
+			<Button appearance="neutral" className="active" soft>
+				Neutral soft
+			</Button>{' '}
+			<Button appearance="faint" className="active" soft>
+				Faint soft
+			</Button>
+		</p>
+
+		<hr />
+
+		<h3>
+			Active buttons with an <code>&lt;a&gt;</code> tag
+		</h3>
+		<p>
+			<Button href="#0" appearance="primary" className="active">
```

Comment:
Is there ever a use case to be able to force a an active state on a button? I don't see the value of it and not a fan of targeting a specific class on the component when using emotion :nail_care:

Reply:
Possibly when a button is used within a dropdown component? 

It's also provided so we can easily illustrate the active styling in the dev/example environment, and run Cypress integration testing over it